hack/lib/images.bash

```diff
@@ -29,15 +38,27 @@ function default_serverless_operator_images() {
   local ocp_version
   local serverless_registry="${registry_quay}/serverless"
 
-  export SERVERLESS_KNATIVE_OPERATOR=${SERVERLESS_KNATIVE_OPERATOR:-$(latest_registry_redhat_io_image_sha "${serverless_registry}-kn-operator:${CURRENT_VERSION_IMAGES}")}
-  export SERVERLESS_OPENSHIFT_KNATIVE_OPERATOR=${SERVERLESS_OPENSHIFT_KNATIVE_OPERATOR:-$(latest_registry_redhat_io_image_sha "${serverless_registry}-openshift-kn-operator:${CURRENT_VERSION_IMAGES}")}
-  export SERVERLESS_INGRESS=${SERVERLESS_INGRESS:-$(latest_registry_redhat_io_image_sha "${serverless_registry}-ingress:${CURRENT_VERSION_IMAGES}")}
-  export SERVERLESS_MUST_GATHER=${SERVERLESS_MUST_GATHER:-$(latest_registry_redhat_io_image_sha "${serverless_registry}-must-gather:${CURRENT_VERSION_IMAGES}")}
+  # When we bump the metadata for a major version, images are not yet available for SO components on
+  # the Konflux registry, so we use the `:latest` tag temporarily by passing "true" as argument to
+  # various `latest_*` functions.
+  #
+  # Eventually, once images are available, SHAs will be replaced/used.
+
+  export SERVERLESS_KNATIVE_OPERATOR=${SERVERLESS_KNATIVE_OPERATOR:-$(latest_registry_redhat_io_image_sha "${serverless_registry}-kn-operator:${CURRENT_VERSION_IMAGES}" "true")}
```

Comment:
Small suggestion: I think we can omit the `:${CURRENT_VERSION_IMAGES}` from all lines where we call `latest_registry_redhat_io_image_sha`. The tag is removed anyway and it's confusing to pass it when we remove it in all cases. Same for `latest_konflux_image_sha`.

Reply:
We can do it in a follow-up.

Reply:
Good point

Reply:
I'll do in a follow up